src/AgConfigure.cpp

```diff
@@ -972,13 +1186,15 @@ int Configuration::getLastOta(void) {
     logError("Current year " + String(curYear) + String(" invalid"));
     return -1;
   }
+  double _t = jconfig[jprop_lastOta];
+  time_t lastOta = (time_t)_t;
```

Comment:
I think we can avoid to work with time values for the OTA feature, and use some measurement cycle counter instead.

Reply:
It's make sure after special period device can trigger OTA again if failed. if you're use the counter it's reset after device reset, for multiple OTA can make damage to FLASH and device can not work any more.